gradio/events.py

```diff
@@ -326,20 +365,26 @@ def stop(
             inputs: List of inputs
             outputs: List of outputs
             api_name: Defining this parameter exposes the endpoint in the api docs
+            status_tracker: StatusTracker to visualize function progress
```

Comment:
Do we have any examples of setting `status_tracker` ? Wondering if it will confuse users.

Reply:
Yeah good point, @aliabid94 maybe it would be good to have an example or demo with `StatusTracker`, since I'm not sure how it's supposed to be used myself

Reply:
No there's not really a good demo for it, I want to reconsider even having this argument.

Reply:
blocks_xray does use it I believe